.github/DEVELOPMENT.md

```diff
@@ -92,6 +94,16 @@ These are tests that will run on an actual device
 - *Essentials.DeviceTests*: Visual Runner running all the .NET MAUI essentials xunit tests.
 - *MauiBlazorWebView.DeviceTests*: Visual Runner for BlazorWebView tests. 
 
+### UI Test Projects
+
+These are tests used for exercising the UI through accessibility layers to simulate user interactions
+
+```
+├──  Controls
+│    ├── tests
+│    │   ├── UITests
```

Comment:
Is this hierarchy accurate?

Should this be `controls.AppiumTests` ?

Reply:
For folder structure, this is accurate. If you are referring to project file then yes it would be Controls.AppiumTests